distribution/tools/server-cli/src/main/java/org/elasticsearch/server/cli/APMJvmOptions.java

```diff
@@ -8,6 +8,7 @@
 
 package org.elasticsearch.server.cli;
 
+import org.apache.logging.log4j.util.Strings;
```

Comment:
Trivial, it might be more straightforward not depend upon log4j here. Maybe use `org.elasticsearch.common.Strings::isNullOrBlank == false` ?